actions/utils.js

```diff
@@ -122,12 +135,86 @@ function getDocPathFromUrl(url) {
     return `${path}.docx`;
 }
 
+async function updateStatusToStateLib(storeKey, status, statusMessage, activationId, action) {
+    const logger = getAioLogger();
+    try {
+        getStatusFromStateLib(storeKey).then((result) => {
+            if (result?.action) {
+                const storeValue = result;
+                if (status) {
+                    storeValue.action.status = status;
+                }
+                if (statusMessage) {
+                    storeValue.action.message = statusMessage;
+                }
+                if (activationId) {
+                    storeValue.action.activationId = activationId;
+                }
+                logger.info(`222 Updating status to state store  -- value :   ${JSON.stringify(storeValue)}`);
```

Comment:
Looks like the prefix `222` `333` etc. were added for local testing ? Should these be removed?